packages/mui-utils/src/clamp/clamp.ts

```diff
@@ -0,0 +1,13 @@
+function clamp(
+  val: number,
+  min: number = Number.MIN_SAFE_INTEGER,
+  max: number = Number.MAX_SAFE_INTEGER,
+): number {
+  if (val == null) {
+    return min;
+  }
```

Comment:
```suggestion
```
@Kamino0 I think we shouldn't have this condition, and instead ensure the `val` is a number before passing it to clamp

Reply:
Made this only in `useSlider`. Should we make same in other clamp usage too? Previously `if (val == null)` was only in `useSlider`'s clamp implementation.